cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/resource/ZookeeperSetOperator.java

```diff
@@ -54,4 +101,224 @@ public static boolean needsRollingUpdate(StatefulSetDiff diff) {
         }
         return false;
     }
+
+    @Override
+    public Future<Void> maybeRollingUpdate(StatefulSet ss, Predicate<Pod> podRestart) {
+        String namespace = ss.getMetadata().getNamespace();
+        String name = ss.getMetadata().getName();
+        final int replicas = ss.getSpec().getReplicas();
+        log.debug("Considering rolling update of {}/{}", namespace, name);
+        Future<Void> f = Future.succeededFuture();
+        boolean zkRoll = false;
+        ArrayList<Pod> pods = new ArrayList<>();
+        String cluster = ss.getMetadata().getLabels().get(Labels.STRIMZI_CLUSTER_LABEL);
+        for (int o = 0; o < replicas; o++) {
+            Pod pod = podOperations.get(ss.getMetadata().getNamespace(), KafkaResources.zookeeperPodNameSufix(name, o));
```

Comment:
Note here, where we use the `zookeeperPodNameSuffix()` method, we do actually have a cluster name to hand, so maybe we don't even need `zookeeperPodNameSuffix()`.